redisinsight/ui/src/components/item-list/ItemList.tsx

```diff
@@ -21,7 +21,8 @@ export interface Props<T> {
   editedInstance: Nullable<T>
   columnVariations: EuiBasicTableColumn<T>[][]
   onDelete: (ids: T[]) => void
-  onExport: (ids: T[], withSecrets: boolean) => void
+  disableExport?: boolean
```

Comment:
Maybe better to cal it hideExport or hideExportBtn. In my opinion, disable - we still have to display it

Reply:
agreed